### ISSUE TITLE
Windows: Crash when create new task

### ISSUE THREAD

**Issue by Enigma7995** [KDAB/Charm#203]
The application crashes after I give the new task a name and click OK.  I used the latest windows installer.  Here is the error information.  
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	Charm.exe
  Application Version:	0.0.0.0
  Application Timestamp:	5513e7bb
  Fault Module Name:	QtCore4.dll
  Fault Module Version:	4.8.5.0
  Fault Module Timestamp:	51cbf354
  Exception Code:	c0000005
  Exception Offset:	0010c6a8
  OS Version:	6.1.7601.2.1.0.768.3
  Locale ID:	1033
  Additional Information 1:	0a9e
  Additional Information 2:	0a9e372d3b4ad19135b953a78882e789
  Additional Information 3:	0a9e
  Additional Information 4:	0a9e372d3b4ad19135b953a78882e789

**Comment by milianw** [comment]
Can you obtain a backtrace somehow? This information is not very useful so far. Is the problem reproducible?

**Comment by Enigma7995** [comment]
I have not downloaded the source code.  I ran into the error by simply using the installer and running the application.  The problem is reproducible and happens every time I go to make a new task.  I used Visual Studio and attached a debugger.  Here is  the call stack if it helps but in the meantime I'm going to download the source and see if the problem still occurs after I build it on my machine.

![call stack](https://cloud.githubusercontent.com/assets/7650201/7849112/eb7216e2-049f-11e5-9f3d-f7eafc999e92.png)

**Comment by Enigma7995** [comment]
I have not had time to come back to the application until now.  I realized that I had been using the wrong method to create a task the whole time.  When charm opened for the first time, I went to the "Select Task" drop down and clicked "Start Other Task...".  Then I tried to type in a task and click "OK".  That is when the application crashed reproducibly.  From this I see two problems...

1.  This is a UI design issue.  The first thing a user is going to do is create a task and the current "Window" -> "Show Task Editor Window" method is  unintuitive for a new user.  Creating the first task should be upfront and obvious.  
Also I don't understand the reason for a task creation window and a separate task selection window.  I think they should be the combined.  What if I want to select a new task but i'm not sure if I have created it yet.  I might go into the task selection window to discover its not there.  Then exit out of the selection window to go to the task creation window only to go back to the selection window.  This should all be together IMHO.

2.  The error that occurs when a user with no tasks goes into the "Select Task..." menu, types in a task and clicks "OK" should not result in a program crash.  This could be handled by combining the select task menu and create task menu.  Otherwise, it should be handled gracefully.

I'm interested in working on this issue but I would like some input from the project maintainers.
